modules/install-sno-supported-cloud-providers-for-single-node-openshift.adoc

```diff
@@ -0,0 +1,19 @@
+// This module is included in the following assemblies: 
+//
+// installing/installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: REFERENCE
+[id="supported-cloud-providers-for-single-node-openshift_{context}"]
+= Supported cloud providers for {sno}
+
+The following table contains a list of supported cloud providers and CPU architectures.
+
+.Supported cloud providers
+[options="header"]
+|====
+|Cloud provider |CPU architecture
+|Amazon Web Service (AWS)|x86-64 and ARM
+|Microsoft Azure|x86-64
+|Google Cloud Platform (GCP) | x86-64 and ARM
```

Comment:
In this case, and for AWS, I think you can also use [this format if you want](https://redhat-documentation.github.io/supplementary-style-guide/#_numbers)
```suggestion
|Google Cloud Platform (GCP) | 64-bit x86 and 64-bit ARM
```
up to you! whichever you prefer.